app/src/main/res/values/colors.xml

```diff
@@ -117,4 +119,4 @@
   <color name="avatar_background_22">#AB29CC</color>
   <color name="avatar_background_23">#CC29B1</color>
   <color name="avatar_background_24">#CC2970</color>
-</resources>
+</resources>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done